pkgs/os-specific/darwin/libcxx/default.nix

```diff
@@ -0,0 +1,26 @@
+{
+  apple-sdk,
+  stdenvNoCC,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "libcxx";
+  inherit (apple-sdk) src version;
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p "$out/include" "$out/lib"
+    cp -v usr/lib/libc++abi.tbd usr/lib/libc++.1.tbd "$out/lib"
+    ln -s libc++.1.tbd "$out/lib/libc++.tbd"
+    cp -rv usr/include/c++ "$out/include"
+    runHook postInstall
+  '';
+
+  passthru = {
+    abiNamespace = "__1";
+    isLLVM = true;
+  };
```

Comment:
These are currently unused, right?

Reply:
They’re leftover from the libc++abi patchset. I can drop this part.

Reply:
The `abiNamespace` was unused, but `isLLVM` is used by `cc-wrapper` to determine if the C++ library is from LLVM or not. Dropping it breaks the wrapper, so I only dropped the ABI namespace.